mbedtls/src/pk/mod.rs

```diff
@@ -139,6 +142,52 @@ impl Pk {
         Ok(ret)
     }
 
+    pub fn generate_ec_using_curve<F: ::rng::Random>(
```

Comment:
See comment on the internal PR for the API i'd like to see here

Reply:
@jethrogb Made those changes but I am running into this
```error[E0277]: `?` couldn't convert the error to `error::Error`
   --> mbedtls/src/pk/mod.rs:134:51
    |
134 |             let curve : EcGroup = curve.try_into()?;
    |                                                   ^ the trait `std::convert::From<impl Into<::Error>>` is not implemented for `error::Error`
    |
    = note: the question mark operation (`?`) implicitly performs a conversion on the error value using the `From` trait
    = help: consider adding a `where error::Error: std::convert::From<impl Into<::Error>>` bound
    = note: required by `std::convert::From::from`
```
and I'm not sure how to fix it.